kong/types.go

```diff
@@ -5,6 +5,16 @@ import (
 	"strings"
 )
 
+type Kong struct {
+	Version     string
+	Enterprise  bool
+	Database    string
+	Credentials struct {
+		hasTagSupport bool
+		minVersion    string
```

Comment:
I created this field to let people know which version is required to have tag support for credentials